lib/rubygems/commands/web_command.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'rubygems/command'
+require 'rubygems/executor'
+require 'rubygems/version_option'
+
+class Gem::Commands::WebCommand < Gem::Command
+
+  include Gem::VersionOption
+  attr_reader :executor
```

Comment:
Is this line used?

Reply:
Nope, removing it!

Reply:
Whoops. Actually I do use this. I use it in `execute` method.

Reply:
@MatheusRich you don't need that `attr_reader` to use `@executor` in the `execute` method

Reply:
@bronzdoc Right. Now how can I stub `fetch_remote_spec`, because I was doing

```ruby
@cmd.executor.stub :fetch_remote_spec, nil
```

Since there's no access to this instance anymore, is there any other way to stub this? I think minitest doesn't have a `stub_any_instance` method.